BUILD.md

```diff
@@ -0,0 +1,137 @@
+# Hyperledger Cactus Build Instructions
+
+This is the place to start if you want to give Cactus a spin on your local
+machine or if you are planning on contributing.
+
+> This is not a guide for `using` Cactus for your projects that have business logic
+> but rather a guide for people who want to make changes to the code of Cactus.
+> If you are just planning on using Cactus as an npm dependency for your project,
+> then you might not need this guide at all.
+
+The project uses Typescript for both back-end and front-end components.
```

Comment:
I thought frontend (Web-application for issuing REST operations) was not in the scope of Cactus.

This statement should be revised with some text like:
"The code written in Typescript in this project requires developers to compile them into Javascript for runtime."

Reply:
> I thought frontend (Web-application for issuing REST operations) was not in the scope of Cactus.

We've been saying the opposite of that for months. A great deal of effort went into the new arch providing this ability to have the sdk package run the same way on both NodeJS and in the browser and for both to have type assist from the compiler to enhance developer productivity. Also the whole build system as it is, provides the ability for the package authors to cherry pick what modules they want to export on the different platforms (browser or NodeJS) so that the packages do not contain the bloat of one platform on the other (for example the NodeJS file system API is not something that should be present in the bundle that will be running in a front end web application).